app/javascript/packs/MainApp/components/RefereeProfile.jsx

```diff
@@ -178,7 +233,8 @@ class RefereeProfile extends Component {
     }
 
     return (
-      <Fragment>
```

Comment:
More love for &lt;Fragment>s! 💔

Reply:
I love them too, but because this is all changing in a separate PR i'd prefer to just leave as is.